### ISSUE TITLE
Get the properties of a window (currently active or address specific) using hyprctl

### ISSUE THREAD

**Issue by xenomorph1096** [hyprwm/Hyprland#3428]
### Description

Using `hyprctl setprop` it is possible to set properties such as transparency, blur, etc for a specific window.

It would be interesting to also be able to retrieve the current value of the property for a specific window, by specifying the address (or using the active window's address by default). This would be useful, to create a graphical options menu (using somthing like eww). `hyprctl getoption` will retrieve the property value set in the hyprland configuration file, but not rules-based properties applied by hyprland at runtime based on a regex on window class or name.

^^ I am actually developing a modal settings menu like this and it would really benefit from being able to initialize the various properties to the current value, rather than the default value read from the config file.

**Comment by matt1432** [comment]
This would be cool but you can already get all that info from `hyprctl clients -j` and use jq to find what you're looking for

**Comment by xenomorph1096** [comment]
That's true, however the set of properties and options are not as complete as I would like. For instance, the information I get for my current window is: 
```json
{
    "address": "0x559d7c727c30",
    "mapped": true,
    "hidden": false,
    "at": [1063, 66],
    "size": [835, 992],
    "workspace": {
        "id": 8,
        "name": "8"
    },
    "floating": false,
    "monitor": 0,
    "class": "firefox",
    "title": "Get the properties of a window (currently active or address specific) using hyprctl · Issue #3428 · hyprwm/Hyprland · GitHub — Mozilla Firefox",
    "initialClass": "firefox",
    "initialTitle": "Mozilla Firefox",
    "pid": 3414,
    "xwayland": false,
    "pinned": false,
    "fullscreen": false,
    "fullscreenMode": 0,
    "fakeFullscreen": false,
    "grouped": [],
    "swallowing": "0x0"
}
```

This would allow me to have a nice menu for toggling fullscreen/floating, and so on, but stuff like the current transparency value, blur, etc is missing.

**Comment by MightyPlaza** [comment]
you can use `hyprctl getoption` to get the values set in the config
and `hyprctl setprop` to set to each window

don't think there is a way to get those values for each window yet

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
